prog/weave-npc/main.go

```diff
@@ -56,7 +57,7 @@ func resetIPTables(ipt *iptables.IPTables) error {
 }
 
 func resetIPSets(ips ipset.Interface) error {
-	// Remove ipsets prefixed `weave-` only
+	// Remove ipsets prefixed `weave-` excluding weave-no-masq-local (used by weaver).
```

Comment:
should we just prefix it something different?  `weaver-no-masq-local` ?  Too subtle?

Reply:
I didn't want to introduce a new prefix, but having the new prefix is probably better than the if clauses.